test/CMakeLists.txt

```
@@ -31,81 +33,81 @@ function( copy_test_file TEST_FILE_NAME )
     configure_file(${CMAKE_CURRENT_LIST_DIR}/testfiles/${TEST_FILE_NAME} ${CMAKE_CURRENT_BINARY_DIR}/test/testfiles/${TEST_FILE_NAME} COPYONLY)
 endfunction()
 
-add_t8_test( NAME t8_gtest_cmesh_bcast       SOURCES t8_gtest_main.cxx t8_cmesh/t8_gtest_bcast.cxx )
-add_t8_test( NAME t8_gtest_eclass            SOURCES t8_gtest_main.cxx t8_gtest_eclass.cxx )
-add_t8_test( NAME t8_gtest_vec               SOURCES t8_gtest_main.cxx t8_gtest_vec.cxx )
-add_t8_test( NAME t8_gtest_mat               SOURCES t8_gtest_main.cxx t8_gtest_mat.cxx )
-add_t8_test( NAME t8_gtest_refcount          SOURCES t8_gtest_main.cxx t8_gtest_refcount.cxx )
-add_t8_test( NAME t8_gtest_occ_linkage       SOURCES t8_gtest_main.cxx t8_gtest_occ_linkage.cxx )
-add_t8_test( NAME t8_gtest_version           SOURCES t8_gtest_main.cxx t8_gtest_version.cxx )
-add_t8_test( NAME t8_gtest_basics            SOURCES t8_gtest_main.cxx t8_gtest_basics.cxx )
-add_t8_test( NAME t8_gtest_netcdf_linkage    SOURCES t8_gtest_main.cxx t8_gtest_netcdf_linkage.cxx )
-add_t8_test( NAME t8_gtest_vtk_linkage       SOURCES t8_gtest_main.cxx t8_gtest_vtk_linkage.cxx )
+add_t8_test( NAME t8_gtest_cmesh_bcast_serial      SOURCES t8_gtest_main.cxx t8_cmesh/t8_gtest_bcast.cxx )
```

We broadcast the cmesh to other procs
```suggestion
add_t8_test( NAME t8_gtest_cmesh_bcast_parallel      SOURCES t8_gtest_main.cxx t8_cmesh/t8_gtest_bcast.cxx )
```

---

Ups :D